pkgs/development/libraries/gperftools/default.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
   '';
 
   NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.isDarwin
```

Comment:
```suggestion
  NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin
```

stdenv.lib is deprecated.

Reply:
OK, I'd fix all the places in this file at once.  Can you link the deprecation?

Reply:
https://github.com/NixOS/nixpkgs/commit/c06b2b3d671da4847f950e7e9041920b40eca0bf

You can search that yourself.

Reply:
Thanks :-)